src/templates/array/ArrayFieldTemplate.js

```diff
@@ -82,7 +82,11 @@ const ArrayFieldTemplate = (props :ArrayFieldTemplateProps) => {
       <ArrayList
           key={`array-item-list-${idSchema.$id}`}>
         {items && items.map(itemProps => (
-          <DefaultArrayItem {...itemProps} orderable={orderable} showIndex={showIndex} />
+          <DefaultArrayItem
+              key={`array-item-${idSchema.$id}-${itemProps.index}`}
```

Comment:
is flow yelling at you not to use an `index` in your key?

Reply:
No. This is because `itemProps.index` is not from the `map` but rather from the property of the item. They do, however, happen to be the same.

Reply:
oops didn't catch that distinction at first. makes sense!